src/libraries/Microsoft.Extensions.Hosting.Systemd/src/ISystemdNotifier.cs

```diff
@@ -3,19 +3,14 @@
 
 namespace Microsoft.Extensions.Hosting.Systemd
 {
-    /// <summary>
-    /// Provides support to notify systemd about the service status.
-    /// </summary>
+    /// <summary>Provides support to notify systemd about the service status.</summary>
```

Comment:
We shouldn't change the format of summary elements for the sake of using a migration tooling. cc @stephentoub 

@carlossanlop can you please re-format to use the previous style?